backend/index.js

```diff
@@ -45,9 +47,11 @@ app.use(setupRedirects())
 app.use(setupStaticContent())
 app.use(setupWebSession())
 app.use(setupAuth({ oauthApi: apis.oauthApi, tokenVerificationApi: apis.tokenVerificationApi }))
-app.use(setupWebpackForDev())
 
 app.use(currentUser({ prisonApi: apis.prisonApi, oauthApi: apis.oauthApi }))
+
+if (!config.app.disableWebpack) app.use(setupWebpackForDev())
```

Comment:
any reason for moving below the `currentUser` stuff - doesn't this mean that the webpack assets will get the current user in dev?

Reply:
No reason really. Technically a webpack module could (I think) make use of req.session, however this is just for dev so I'm not too worried.